src/flowlayout.cpp

```diff
@@ -38,7 +38,11 @@
 **
 ****************************************************************************/
 
+#if (QT_VERSION < 0x050000)
```

Comment:
I'm forgot that in readme I mention the Qt min. requirement is 4.6, but currently I'm developing at > 5, silly me :stuck_out_tongue: